Misc/NEWS.d/next/Library/2019-12-10-21-27-37.bpo-39019.7o43Gq.rst

```diff
@@ -0,0 +1 @@
+Implement ``__class_getitem__`` for ``http.cookies.Morsel`.
```

Comment:
The test are failing because you have unbalanced quotes in http.cookies.Morsel

Reply:
Thanks for the catch Pablo.